.changeset/every-schools-find.md

```diff
@@ -0,0 +1,20 @@
+---
+'@backstage/plugin-catalog-unprocessed-entities': patch
+'@backstage/frontend-app-api': patch
+'@backstage/core-compat-api': patch
+'@backstage/plugin-scaffolder-react': patch
+'@backstage/plugin-catalog-import': patch
+'@backstage/plugin-notifications': patch
+'@backstage/plugin-kubernetes': patch
+'@backstage/plugin-scaffolder': patch
+'@backstage/plugin-api-docs': patch
+'@backstage/plugin-devtools': patch
+'@backstage/plugin-techdocs': patch
+'@backstage/plugin-catalog': patch
+'@backstage/plugin-signals': patch
+'@backstage/plugin-search': patch
+'@backstage/plugin-home': patch
+'@backstage/plugin-app': patch
+---
+
+Internal update to use the new variant of `ApiBlueprint`.
```

Comment:
Nit: 👀 s/"internal"/"core"?

Reply:
It's referring to an internal update of the package, just code being updated without affecting the external API or behavior of the package